docs/UserGuide.adoc

```diff
@@ -102,7 +102,33 @@ Deletes the second person in the person list.
 * `del 4` +
 Deletes the fourth person in the person list.
 
-=== Find a group / person / meeting: `find`
+=== Adding a new group: `addGroup`
+
+Add a new group with user input title into MeetingBook. +
+
+Format: `addGroup n/[Name]` +
+
+Examples:
+
+* `addGroup n/CS2103T` +
+Added a new group with title 'CS2103T' into the MeetingBook.
```

Comment:
Should be in present tense, e.g. Adds a new group with title 'CS2103T' into the MeetingBook.
Same for other examples.

Reply:
Will correct them soon!